static/js/main.js

```diff
@@ -1,15 +1,7 @@
-let toggleAdvancedFields = document.getElementById("toggle-advanced-fields");
-if (toggleAdvancedFields) {
-  toggleAdvancedFields.addEventListener("click", function () {
-    const advancedFields = document.getElementById("advanced-fields");
-    if (advancedFields.style.display === "none") {
-      advancedFields.style.display = "block";
-      this.textContent = "Hide Advanced Fields";
-    } else {
-      advancedFields.style.display = "none";
-      this.textContent = "Show Advanced Fields";
-    }
-  });
+function toggleAdvancedFields(el) {
+  const advancedFields = document.getElementById("advanced-fields");
+  const isHidden = advancedFields.hidden = !advancedFields.hidden;
+  el.textContent = isHidden ? "Show Advanced Fields" : "Hide Advanced Fields";
```

Comment:
@henry-r18 I like the changes to this, making it cleaner and more modern.  It works well on the `edit_item` screen... but unfortunately, this change has broken the toggle on the `view_item` screen.

For example, see http://127.0.0.1:8000/view_item/6/
Clicking "Show Advanced Fields" now does nothing.

I think the function is fine, but the `view_item` template should be updated as needed to work with the new javascript.